pkg/catalog/catalog.go

```diff
@@ -124,12 +124,9 @@ func IsDevfileComponentSupported(devfile Devfile) bool {
 			hasRunCommand = strings.Contains(command.Name, "devRun")
```

Comment:
Should we make this case insensitive? 🤔 

push validation doesnt differentiate between `devRun` or `devrun`

Reply:
Yes, it should be case insensitive